src/Resources/config/services.yml

```diff
@@ -1,7 +1,7 @@
 services:
     partnermarketing_file_system.factory:
-        class: "Partnermarketing\FileSystemBundle\Factory\FileSystemFactory"
+        class: "Partnermarketing\\FileSystemBundle\\Factory\\FileSystemFactory"
```

Comment:
Was this not working before? If it was I don't think we should double up on backslashes without reason